recognition/include/pcl/recognition/impl/ransac_based/simple_octree.hpp

```diff
@@ -8,392 +8,365 @@
 #ifndef SIMPLE_OCTREE_HPP_
 #define SIMPLE_OCTREE_HPP_
 
-//===============================================================================================================================
+#include <algorithm>
+#include <cmath>
 
-template<typename NodeData, typename NodeDataCreator, typename Scalar> inline
-pcl::recognition::SimpleOctree<NodeData, NodeDataCreator, Scalar>::Node::Node ()
-: data_ (nullptr),
-  parent_ (nullptr),
-  children_(nullptr)
-{}
-
-//===============================================================================================================================
-
-template<typename NodeData, typename NodeDataCreator, typename Scalar> inline
-pcl::recognition::SimpleOctree<NodeData, NodeDataCreator, Scalar>::Node::~Node ()
+namespace pcl
```

Comment:
Other files don't have this change. Maybe restrict namespace addition to the existing PR.

Reply:
This came out of a conflict whilst cherry-picking. I couldn't understand what was happening, now I do, thanks!